src/View/Helper/HtmlHelper.php

```diff
@@ -344,7 +344,14 @@ public function link($title, $url = null, array $options = [])
             $title = h(urldecode($title));
             $escapeTitle = false;
         }
-
+        
+        /* A "simple phishing" vulnerability
+         * Consider adding a configuration option to disable?
+         */ https://dev.to/ben/the-targetblank-vulnerability-by-example
+        if (isset($options['target'])) {
+            $options['rel'] = 'noopener noreferrer';
+        }
+        
```

Comment:
My thoughts of fixing this would be adding an option in the link such as `"targetProtect" => false` With this, I feel it best if I make a private function to handle this, inside of the HtmlHelper. Would that be appropriate or should I go about it in a different manner?